config/vufind/searches.ini

```
@@ -709,6 +709,12 @@ enabled = true
 default_handler = Solr
 ; Auto-submit autocomplete on click or enter
 auto_submit = true
+; Maximum number of suggestions shown. This will limit the number of items
+; in the drop down list created by the autocomplete javascript library. It
+; does not have any impact on the number of suggestions returned by
+; the server-side autocomplete handler code. This number may vary from
```

I realize I'm reviewing my OWN code now, but on re-reading this, I think it's better if we make this change:

```suggestion
; the server-side autocomplete handler code, which may vary from
```